app/controllers/miq_ae_customization_controller.rb

```diff
@@ -257,13 +257,14 @@ def first_sub_node_is_a_folder?(node)
     sub_node == "xx" || !(sub_node == "" && node.split('_').length <= 2)
   end
 
-  def get_node_info
+  def get_node_info(node = {}, _show_list = true)
```

Comment:
I don't think this second argument should be there, since you've already changed all the places that call this method. I'd prefer having something like `attribute_accessor_with_default :show_list, true) and set it on the callers.

That is, if you found and changed all callers, if not (or if there's no obvious way to find them), you can disregard this comment and the current implementation is fine.